### PR TITLE
chore: use default parameters replace defaultProps

### PR DESCRIPTION
**Pull request by li-jia-nan**
- ref: #5839

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ interface ButtonProps {
 export function Button({
   children,
   onClick,
-  active,
+  active = false,
   className,
-  style,
+  style = {},
```
Comment (gaearon) [inline src/components/Button.tsx:21]: Just remove it — it's fine to have it be `undefined`.

Comment (li-jia-nan) [reply src/components/Button.tsx]: done~

Comment (li-jia-nan) [reply src/components/Button.tsx]: pls review，thx~

**Comment by li-jia-nan** [comment]
I don't know why it prompts me `Deployment has failed`, what can I do for this?

**Comment by li-jia-nan** [comment]
image

**Comment by li-jia-nan** [comment]
I don't know why ci fail, Is there anything I can do for this ?

**Pull request closed without merging**